soc/intel/intel_adsp/ace/Kconfig.soc

```diff
@@ -8,7 +8,7 @@ config SOC_SERIES_INTEL_ACE
 	  Intel ADSP ACE
 
 config SOC_SERIES
-	default "ace" if SOC_SERIES_INTEL_ACE
+	default "intel_ace" if SOC_SERIES_INTEL_ACE
```

Comment:
should be intel_adsp_ace

Reply:
then `SOC_SERIES_INTEL_ACE` needs to be `SOC_SERIES_INTEL_ADSP_ACE` as well

Reply:
yes

Reply:
> yes

@nashif even if we need to align at SOF as well ?

Reply:
> should be intel_adsp_ace

@nashif, renamed as suggested